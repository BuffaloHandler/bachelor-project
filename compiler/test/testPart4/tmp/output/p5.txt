0
1
123
0
1
0
0
28
