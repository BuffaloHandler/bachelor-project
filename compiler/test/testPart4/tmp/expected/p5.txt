0
1
123
4
0
1
0
0
28
ret 0
